virtual-setup/rstudio-login.md

```diff
@@ -1 +1,56 @@
-## RStudio Server instructions
\ No newline at end of file
+## RStudio Server Set Up
+
+We will send you your Username and temporary password by a direct message on Slack, after you are logged in on [Cancer Data Science](http://ccdatalab.org/slack).
+
+Go to https://rstudio.ccdatalab.org (You may want to bookmark this) and type in your username and temporary password, and click `Sign in`.
+
+![RStudio Login](screenshots/rstudio-server-login.png)
+
+Signing in should bring you to the RStudio session page.
+Click on the `Terminal` tab.
+
+![RStudio Session](screenshots/rstudio-session-terminal.png)
+
+Type in the `passwd` command in the `Terminal` tab.
+
+![RStudio change password](screenshots/rstudio-password-change-1.png)
+
+...and press `Enter`. Then type in the password you were given in Slack.
+
+![RStudio change password](screenshots/rstudio-password-change-2.png)
+
+Type in the new password you've chosen once, and press `Enter`, then it will ask you to confirm by typing it in again.
+Also press `Enter`.
+If you forget your password at anytime, Slack on of the CCDL team members to assist you.
+
+## Navigating the RStudio Server
+
+In the upper right corner of the session page, you should see these buttons:
+
+![RStudio Navigation](screenshots/rstudio-session-buttons.png)
+
+You can restart your R session with the orange, circular, on/off button.
```

Comment:
```suggestion
You can restart your R session with the orange circular on/off button.
```

Reply:
I feel like the behavior of this button may be inconsistent. Whenever I click in on the server I get sent to the workspaces page. It never just restarts the server for me.

Reply:
Love this for us 😂

Reply:
Are we talking about the same thing @jashapiro : https://github.com/AlexsLemonade/training-modules/pull/194#discussion_r413946105

Reply:
I find that sometimes it sends me to the workspaces page, sometimes it leaves me at the `Session Ended` page with the guy going out the door.

Reply:
Currently, it is sending me directly to the workspaces page, so that's what I'm writing it as.

Reply:
I have literally never seen
> Session Ended page with the guy going out the door.

Reply:
> Are we talking about the same thing @jashapiro : [#194 (comment)](https://github.com/AlexsLemonade/training-modules/pull/194#discussion_r413946105)

Yes, I think so. So I tend to think "restart" is the wrong verb in this sentence.

Reply:
This guy, going out the door:
![r-studio-session-ended](https://user-images.githubusercontent.com/23458084/80131640-86a0ce80-8568-11ea-92b7-464b39bb267c.png)

Reply:
Ah, that guy.